backend/src/project/structures/Variable.ts

```diff
@@ -1,7 +1,7 @@
 import { Static, Record, String, Number } from "runtypes";
 
 export const Variable = Record({
-    id: String,
+    id: String.withConstraint((s) => s.split("@").length == 2),
```

Comment:
`s.split("@").length == 2` powinno dać sie zastąpić `s.includes("@")`

Reply:
Uważam, że `s.split("@").length == 2` bardziej weryfikuje, bo odrzuci przykładowo `abc@` albo `@10`. Celowo dałem splita, żeby za jednym zamachem sprawdzić, czy tekst na lewo i prawo od @ jest niepusty

Reply:
No to tak nie działa jak zakładasz
![image](https://user-images.githubusercontent.com/57133114/221430181-96255968-772e-4243-ba0d-44b6d055aa58.png)